docs/usage.md

```diff
@@ -193,18 +233,18 @@ A location does not have a prescribed shape. In your UI it can represent a singl
 
 The location property will be set on the [member](#members).
 
-Because locations are part of members, a `memberUpdate` event will be emitted when a member updates their location. When a member leaves, their location is set to `null`.
+Because locations are part of members, a member event will be emitted when a member updates their location. When a member leaves, their location is set to `null`.
 
 ```ts
-space.subscribe('membersUpdate', (members) => {
-  console.log(members);
+space.subscribe('update', (memberProfileUpdate) => {
+  console.log(memberProfileUpdate);
```

Comment:
`memberProfileUpdate` looks odd in this example as that's essentially how we refer to arbitrary profile data. Maybe `member` and `member.location`?